tests/common/dualtor/tor_failure_utils.py

```diff
@@ -27,15 +27,16 @@ def shutdown_tor_bgp(duthost, shutdown_all=True):
         bgp_neighbors = duthost.get_bgp_neighbors()
         up_bgp_neighbors = [k.lower() for k, v in bgp_neighbors.items() if v["state"] == "established"]
         if shutdown_all and up_bgp_neighbors:
-            logger.info("Shutdown BGP sessions on {}".format(duthost.hostname))
```

Comment:
Should the name `shutdown_tor_bgp` now be changed to `kill_bgpd`?

Reply:
Yes, will fix